test/suites/dev/moonbase/test-staking/test-staking-locks9.ts

```diff
@@ -8,12 +8,11 @@ import {
   alith,
   generateKeyringPair,
 } from "@moonwall/util";
-import { fromBytes } from "viem";
-import { chunk } from "../../../../helpers";
+import { chunk, getDelegatorStakingFreeze, getNumberOfDelegatorFreezes } from "../../../../helpers";
 
 describeSuite({
-  id: "D023484",
-  title: "Staking - Locks - max delegations",
+  id: "D013482",
```

Comment:
This change was also caused by a conflict, we should revert it to keep the test id order.